### PR TITLE
Update create-a-dlp-policy-from-a-template.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed typo

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6290

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/create-a-dlp-policy-from-a-template.md [hunk 1]
```diff
@@ -81,11 +81,7 @@ You can also choose the Custom template, which has no default rules, and configu
 10. Choose **Next**.
 
 11. On the **Protection actions** page if you want, you can customize the policy tip notifications and notification emails. Enable **When content matches the policy conditions, show policy tips to users and sen them an email notification**, then choose **Customize the tip and email**.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-dlp-policy-from-a-template.md]:
```suggestion
11. On the **Protection actions** page if you want, you can customize the policy tip notifications and notification emails. Enable **When content matches the policy conditions, show policy tips to users and send them an email notification**, then choose **Customize the tip and email**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - Minor additional edit, please.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**